### PR TITLE
Before release

### PR DESCRIPTION
**Pull request by talhasch**

### CODE REVIEW

File: src/common/components/friends/index.tsx [hunk 1]
```diff
@@ -140,13 +140,29 @@ export class List extends Component<ListProps, ListState> {
...
+        const {account, mode} = this.props;
+
+        const follower = (mode === "following" ? account.name : search);
+        const following = (mode === "following" ? search : account.name);
+
+        let connection: boolean;
+        let ac: Account | null = null;
         try {
-            data = await this.fetch(search.replace("@", ""), 1);
+            const resp = await getRelationshipBetweenAccounts(follower, following);
```
Comment (feruzm) [inline src/common/components/friends/index.tsx:151]: Couldn't we just do live search from data[], if not all data[] is loaded, Load more button could stay and by keeping search input we can do search once load more clicked and search from new data[]?

Comment (talhasch) [reply src/common/components/friends/index.tsx:151]: i don't get purpose of this?

Comment (feruzm) [reply src/common/components/friends/index.tsx:151]: Search requiring user to press `enter` and should use exact username, so search looses its purpose somehow. Instead we can do `filter` on array `data[]`. If user didn't load all followers, then `Load more` could keep showing and search continues to filter from what data is available already in memory.

**Comment by talhasch** [comment]
@feruzm implemented api endpoints for friend search

**Comment by talhasch** [comment]
@feruzm added typeahead search. pls check it.

**Comment by talhasch** [comment]
@feruzm pushed workaround for pin/unpin

**Review by feruzm — APPROVED** [review]

**Pull request merged**